angular/demo/src/app/samples/slider/custom.route.ts

```diff
@@ -0,0 +1,74 @@
+import {SliderComponent, SliderHandleDirective, SliderLabelDirective} from '@agnos-ui/angular';
+import {Component, NgZone, inject} from '@angular/core';
+import {FormControl, FormsModule, ReactiveFormsModule} from '@angular/forms';
+import {take} from 'rxjs';
+
+@Component({
+	standalone: true,
+	imports: [SliderComponent, SliderLabelDirective, SliderHandleDirective, ReactiveFormsModule, FormsModule],
+	template: `
+		<h2>Slider with custom labels</h2>
+		<div auSlider auMin="0" auMax="100" auStepSize="1" [formControl]="sliderCustomLabelControl">
+			<ng-template auSliderLabel let-value="value">{{ value }}%</ng-template>
+		</div>
+		<hr />
+
+		<h2>Slider with custom handle</h2>
+		<div auSlider auMin="0" auMax="100" auStepSize="1" [formControl]="sliderCustomHandleControl">
+			<ng-template auSliderHandle let-item="item" let-state="state" let-widget="widget">
+				<button
+					class="au-custom-slider-handle"
+					role="slider"
+					[attr.aria-valuemin]="state.min"
+					[attr.aria-valuemax]="state.max"
+					[attr.aria-readonly]="state.readonly ? true : null"
+					[attr.aria-disabled]="state.disabled ? true : null"
+					[attr.aria-valuenow]="item.value"
+					[attr.aria-valuetext]="item.value"
+					[attr.aria-label]="item.ariaLabel"
+					[attr.aria-orientation]="state.vertical ? 'vertical' : null"
+					[disabled]="state.disabled"
+					[class]="state.vertical ? 'au-custom-slider-handle-vertical' : 'au-custom-slider-handle-horizontal'"
+					[style.left.%]="state.handleDisplayOptions[item.id].left"
+					[style.top.%]="state.handleDisplayOptions[item.id].top"
+					(keydown)="onKeyDown($event, item.id, widget.actions.keydown)"
```

Comment:
So the problem of the focus comes from the fact that the contents of the button is an svg, so the _event.target_ is a path of said svg. We can avoid having the 'custom' methods on the button, if you modify the **mouseDown** core method to use _event.currentTarget_ instead of the _event.target_. It takes the actual element on which we put event and not its contents (so in our case it will be button and you can remove _focusParent_ too). 
Be careful as apparently the currentTarget needs to be assigned straight away to a variable in order to be used, so in mouseDown you need to write: const currentTarget = event.currentTarget immediately, otherwise it will be null :)

Reply:
That's a good suggestion that actually would make it works with any content of the button.
done